docs/building.md

```diff
@@ -0,0 +1,39 @@
+## Building, flashing, and debugging
+
+### Build
+[Here](https://github.com/lpn-plant/ms-tpm-20-ref) you can find our fork of
+[Official TPM 2.0 Reference Implementation (by Microsoft)](https://github.com/microsoft/ms-tpm-20-ref).
+
+The project is developed using STM32CubeIDE so the building process is
+straightforward.
+
```

Comment:
Is there a way to generate Makefile-based project from STM32CubeIDE, as it was possible to generate from the CubeMX?